dartagnan/src/main/java/com/dat3m/dartagnan/program/processing/Intrinsics.java

```diff
@@ -812,4 +1171,50 @@ private List<Event> inlineMemSet(FunctionCall call) {
         return replacement;
     }
 
+    private List<Event> inlineStrcpy(FunctionCall call, boolean returnEnd, boolean numberProvided) {
```

Comment:
Given that this function is not used yet, I think it should be removed from this PR.